From 6a403ce0f15a5a2ac91da2e8ede16c53fe3727ca Mon Sep 17 00:00:00 2001
From: Marcin Wojtas <mw@semihalf.com>
Date: Sun, 14 Feb 2016 00:36:36 +0100
Subject: [PATCH 161/538] arm64: enable CONFIG_MV_XOR in mvebu_v8_lsp_defconfig

Since new Armada 3700 SoC from Marvell uses mv_xor dmaengine,
its config can be enabled.

Change-Id: I7c5eab0ecde7db4fe5d012dd287f7b5f5df64a27
Signed-off-by: Marcin Wojtas <mw@semihalf.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/27676
Reviewed-by: Nadav Haklai <nadavh@marvell.com>
Tested-by: Nadav Haklai <nadavh@marvell.com>
---
 arch/arm64/configs/mvebu_v8_lsp_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/mvebu_v8_lsp_defconfig b/arch/arm64/configs/mvebu_v8_lsp_defconfig
index f34b41f..63ad1a6 100644
--- a/arch/arm64/configs/mvebu_v8_lsp_defconfig
+++ b/arch/arm64/configs/mvebu_v8_lsp_defconfig
@@ -157,6 +157,7 @@ CONFIG_LEDS_TRIGGER_CPU=y
 CONFIG_RTC_CLASS=y
 CONFIG_RTC_DRV_EFI=y
 CONFIG_DMADEVICES=y
+CONFIG_MV_XOR=y
 CONFIG_MV_XOR_V2=y
 CONFIG_DMATEST=y
 CONFIG_VIRTIO_PCI=y
-- 
1.9.1

